<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI_Options</class>
 <widget class="QDialog" name="GUI_Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>481</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="View">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab_view">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>50</y>
       <width>461</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Viewport</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_cameraMovementSpeed">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_view_movementSpeed">
          <property name="text">
           <string>Camera movement speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_view_cameraSpeed">
          <property name="suffix">
           <string/>
          </property>
          <property name="minimum">
           <double>-9999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>9999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_cameraRotationSpeed">
        <item>
         <widget class="QLabel" name="label_view_rotSpeed">
          <property name="text">
           <string>Camera rotation speed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_view_cameraRotSpeed">
          <property name="minimum">
           <double>-9999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>9999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="button_view_backgroundColorSelector">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Change background color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_view_reset">
        <property name="text">
         <string>Reset viewport settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_theme">
      <item>
       <widget class="QLabel" name="label_theme">
        <property name="text">
         <string>UI Theme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_theme"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_export">
    <attribute name="title">
     <string>Export</string>
    </attribute>
    <widget class="QRadioButton" name="radioButton_export_exportBaseDir">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Export to the 'base directory' folder</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_export_exportCustomDir">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Export to an other folder</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_export_copyTexturesSlot0">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Texture0 : Diffuse map</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_export_copyTexturesSlot1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Texture1 : Normals map</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_export_copyTextures">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Copy textures :</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_export_copyTexturesSlot2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Texture2 : Specular map</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>180</y>
       <width>451</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox_export_convertTextures">
        <property name="text">
         <string>Convert textures to another format :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_export_texturesFormat">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>.jpg</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.png</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.tga</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.pcx</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>.ppm</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineEdit_export_exportDir"/>
      </item>
      <item>
       <widget class="QPushButton" name="button_export_selectExportDir">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_tw1">
    <attribute name="title">
     <string>TW1 settings</string>
    </attribute>
    <widget class="QCheckBox" name="checkBox_TW1_loadPaintedMeshes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Load painted meshes (used for high def terrains)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_TW1_loadSkinnedMeshes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Load skinned meshes</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_TW1_loadStaticMeshes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Load static meshes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_TW1_importSettings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Import settings</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_tw2">
    <attribute name="title">
     <string>TW2 settings</string>
    </attribute>
    <widget class="QCheckBox" name="checkBox_TW2_loadBestLOD">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>451</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Load only the best LOD</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_TW2_importSettings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Import settings</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_tw3">
    <attribute name="title">
     <string>TW3 settings</string>
    </attribute>
    <widget class="QLabel" name="label_TW3_texFolder">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Textures folder :</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_TW3_loadSkel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Load skeleton (experimental - disable it if you don't need it)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_TW3_loadBestLOD">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>341</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Load only the best LOD</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_TW3_texFolder"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_TW3_selectTexFolder">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_TW3_importSettings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Import settings</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_debug">
    <attribute name="title">
     <string>Debug</string>
    </attribute>
    <widget class="QCheckBox" name="checkBox_debug_log">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>251</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable &quot;debug.log&quot;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_debug_log">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>441</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>If you have a problem to load a w2ent/w2mesh file, you can enable this option to generate a log file. This file can help me to solve your problem, so in case of problem, you can send me this file (via the forum or by mail at jeanlouis[dot]boudrand[at]hotmail[dot]fr.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
